tests/dag_processing/test_dag_bundles.py

```diff
@@ -144,6 +147,17 @@ def setup_class(cls) -> None:
                 conn_id=CONN_ONLY_PATH,
                 host="path/to/repo",
                 conn_type="git",
+                extra={"key_file": "path/to/key"},
```

Comment:
Adding this sorta defeats the purpose - you don't need a key for a local path :)

Reply:
🙏🏽 Yes, Done!